Frontend/src/data/rental/onSave.js

```diff
@@ -44,44 +29,65 @@ const newItemStatus = (rental) => {
   }
 };
 
-export default async (rental, closePopup, updateItemStatus, createNew) => {
-  setNumericValuesDefault0(rental, columns);
+const updateItemStatus = async (item, status) => {
+  await Database.updateDoc({ ...item, status });
+  await WoocommerceClient.updateItem(item);
+  notifier.success(
+    `'${item.name}' wurde als ${
+      item.status === "instock" ? "verfügbar" : "verliehen"
```

Comment:
Somehow with the new additions, the popup logic gets reserversed:

When renting something it displays "Item.name wurde als verfügbar markiert" and vice versa. Not sure if swapping the conditional will solve this or if there is something more going on

Reply:
No the logic does not get reversed. But you are right, the notification text was reversed, I fixed that.